webapp/src/components/ProgramActivityDialog.tsx

```diff
@@ -76,9 +76,12 @@ const ProgramActivityDialog = ({
           {timeRange()}
         </DialogContentText>
 
```

Comment:
I think we can remove the extra spaces in lines 71, 73, 78, 80, 84, 86 and 88.

Reply:
Agreed! Please always make sure you run `yarn delint` before pushing to GitHub :) thank you